content/blog/2023/unintuitive-objects.md

```diff
@@ -52,19 +60,18 @@ the published date and time,
 the updated date and time,
 the category or topic, and more.
 
-Instances are all the specific occurences
+Instances are all the specific occurrences
 of an object.
 To go back to a physical object,
 instances of the chair object in my house
 would include the pink roller chair I sit in at my desk,
 the small chair my daughter uses in her room,
 the tiny chair in my daughter's dollhouse,
-each chair at the kitchen table.
+and each chair at the kitchen table.
 
 {{ embed.img(
   src='blog/2023/chair-object-shop.jpg',
-  alt='',
-  caption='Five instances of a CHAIR object in a bookshop'
+  alt='Five instances of a CHAIR object in a bookshop'
```

Comment:
I'm not sure if you specifically wanted a caption? It looks like we don't currently have captions for `embed.img`, but we do for `embed.figure`? I'll let @mirisuzanne speak to that.

Reply:
Yeah, I figure is just any number of images (or other media) with a caption. This text does feel more like caption text than alt text – it's providing context for the image more than describing it – but even when there's a caption, it's useful to have alt text. For the alt text, consider how you would describe the image to someone so they know what it looks like to have five instances of a chair object in a bookshop.